requirements.txt

```diff
@@ -1,11 +1,13 @@
 torch
-pytorch-lightning
+lightning
 torchmetrics
+rich
 fair-esm
-jsonargparse
 wandb
 tokenizers
 transformers
 beartype
 datasets
 transformers[torch]
+pytest
+pytest-cov
```

Comment:
Should we start specifying versions to avoid breaking changes and use dependabot to update andreview version changes?

Reply:
yeah, sure

Reply:
This should be easily accomplished by installing requirements and then running `pip freeze`